debian/control

```diff
@@ -1,9 +1,10 @@
 Source: kodi-inputstream-ffmpegdirect
 Priority: extra
 Maintainer: Ross Nicholson <phunkyfish@gmail.com>
-Build-Depends: debhelper (>= 9.0.0), cmake,
-               kodi-addon-dev, pkg-config, libssl-dev, zlib1g-dev, yasm
-Standards-Version: 3.9.6
+Build-Depends: debhelper (>= 9.0.0), cmake, kodi-addon-dev,
+               libavutil-dev (>= 4.2.2), libavcodec-dev (>= 4.2.2), libavformat-dev (>= 4.2.2), libavfilter-dev (>= 4.2.2),
+               libbz2-dev, libp8-platform-dev, libssl-dev, libz3-dev, pkg-config, yasm, zlib1g-dev
+Standards-Version: 3.9.8
```

Comment:
Make it sense to use there also `4.1.2`? 
Mostly all addons have this currently and to hold a bit equal.

Gives it a table which version match witch ubuntu and debian?

Reply:
iirc 4.1.2 is debhelper 11, so we could use that, but it does not really make a difference atm

Reply:
Thought only to have them equal, that it works I seen by older addons.

Was also not sure how is in latest, but have looked now his debhelper changelog.
There is now `4.5.0` as debhelper 12 and except `4.1.4` (where `Beyond the dh_usrlocal change listed above, no changes were required.`) with only a `no changes were required` everywhere.

Therefore, you can leave it as it is or update it, regardless of the behavior of the versions there.